src/neuroglancer/datasource/boss/frontend.ts

```diff
@@ -327,7 +331,7 @@ export class MultiscaleVolumeChunkSource implements GenericMultiscaleVolumeChunk
               collection: this.experimentInfo.collection,
               experiment: this.experimentInfo.key,
               channel: this.channel,
-              resolution: scaleInfo.key,
+              resolution: parseInt(scaleInfo.key, 10) + baseResolution,
```

Comment:
is this defaulting to 10 if it doesn't have scaleInfo.key?  Just curious.

Reply:
No, this is 10 for base 10. Normally, this is the default, but there's a quirk with JS where if the number looks like an octal number, it'll do base 8, instead, if you don't specify the base.